packages/gatsby-theme-docs/src/layouts/internals/layout-release-note-body.js

```diff
@@ -20,8 +21,30 @@ const Topics = styled.div`
   }
 `;
 
+const Product = styled.div`
```

Comment:
The naming is a little misleading. you could use something more generic like ReleaseNotesStamp or CustomStamp